beacon_node/http_api/tests/tests.rs

```diff
@@ -4152,6 +4152,53 @@ impl ApiTester {
         self
     }
 
+    pub async fn test_get_expected_withdrawals_invalid_state(self) -> Self {
+        let state_id = CoreStateId::Root(Hash256::zero());
+
+        let result = self.client.get_expected_withdrawals::<E>(&state_id).await;
+
+        match result {
+            Err(e) => {
+                assert_eq!(e.status().unwrap(), 404);
+            }
+            _ => panic!("query did not fail correctly"),
+        }
+
+        self
+    }
+
+    pub async fn test_get_expected_withdrawals_capella(self) -> Self {
+        let slot = self.chain.slot().unwrap();
+        let state_id = CoreStateId::Slot(slot);
+
+        let result = self.client.get_expected_withdrawals::<E>(&state_id).await;
```

Comment:
I think calling `unwrap` on this result achieves the same thing without the below `match` statement, given we're not making any assertions here - although I wonder if it's possible to verify the withdrawals in the response?

Reply:
I decided to update the test to calculate the expected withdrawal, and then make a request to the client to compare the expected withdrawal calculated vs the withdrawal response that the client returns